files/ko/web/api/baseaudiocontext/createperiodicwave/index.html

```diff
@@ -0,0 +1,155 @@
+---
+title: BaseAudioContext.createPeriodicWave()
+slug: Web/API/BaseAudioContext/createPeriodicWave
+tags:
+  - API
+  - Audio
+  - AudioContext
+  - BaseAudioContext
+  - Media
+  - Method
+  - Reference
+  - Web Audio API
+  - createPeriodicWave
+browser-compat: api.BaseAudioContext.createPeriodicWave
+---
+<p>{{ APIRef("Web Audio API") }}</p>
+
+<p>{{ domxref("BaseAudioContext") }} 인터페이스의 <code>createPeriodicWave()</code> 메서드는 {{domxref("PeriodicWave")}}를 생성하기 위해 사용되는데, 이는 {{ domxref("OscillatorNode") }}의 출력을 형성하기(shape) 위해 사용될 수 있는 주기적인 파형을 정의하기 위해 사용됩니다.</p>
```

Comment:
```suggestion
<p>{{ domxref("BaseAudioContext") }} 인터페이스의 <code>createPeriodicWave()</code> 메서드는 {{domxref("PeriodicWave")}}를 생성하기 위해 사용되는데, 이는 {{ domxref("OscillatorNode") }}의 출력을 형성하기 위해 사용될 수 있는 주기적인 파형을 정의하기 위해 사용됩니다.</p>
```

특수한 경우를 제외하고 (shape) 라고 표기할 정도로 단어의 의미가 명확하지 않다면 (shape) 라고 표기하는것보다 다른 단어를 찾아야한다고 생각합니다. 없어도 괜찮을듯한데 어떻게 생각하시나요?